modules/hal_nordic/nrfx/Kconfig.logging

```diff
@@ -28,6 +28,10 @@ config NRFX_GPIOTE_LOG
 	bool "GPIOTE driver logging"
 	depends on NRFX_GPIOTE
 
+config NRFX_GRTC_LOG
+	bool "GRTC driver logging"
+	depends on NRFX_GRTC
```

Comment:
Surely logging should depend upon the logging system being enabled?

Reply:
The whole menu already depends on `LOG` (at the beginnig of the file).